doc/langdef.md

```diff
@@ -1177,9 +1288,20 @@ Regular expressions follow the [RE2 syntax](https://github.com/google/re2/wiki/S
       map indexing
     </td>
   </tr>
+  <tr>
+    <th rowspan="1">
+      @not_strictly_false
```

Comment:
This is problematic, since ordinary users can't type this in.  Two options.  1) Omit it, either manually or in the dumping tool.  2) Add a comment outside the generated section that the functions starting with "@" aren't part of the abstract language spec.

Reply:
I can leave `@not_strictly_false` out of the docs since it's not publicly visible.